base_access_wallet.sol

```diff
@@ -178,8 +179,6 @@ contract BaseAccessWalletFactory is Ownable {
     bytes32 public version ="BsAccWltFactory20190506154200ML"; //class name (max 16), date YYYYMMDD, time HHMMSS and Developer initials XX
 
     function createAccessWallet() public returns (address) {
-        return  (new BaseAccessWallet(msg.sender));
+        return (new BaseAccessWallet(msg.sender));
```

Comment:
should this be `return(new ExternalUserWallet(msg.sender))`?

Reply:
No - the 'external' wallet is more for future work.

Reply:
oh ok